### PR TITLE
[pornhub] Fix like and dislike count extraction

### PR DESCRIPTION
**Pull request by imba-tjd**
## Please follow the guide below

- You will be asked some questions, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *pull request* (like that [x])
- Use *Preview* tab to see how your *pull request* will actually look like

---

### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [adding new extractor tutorial](https://github.com/ytdl-org/youtube-dl#adding-support-for-a-new-site) and [youtube-dl coding conventions](https://github.com/ytdl-org/youtube-dl#youtube-dl-coding-conventions) sections
- [x] [Searched](https://github.com/ytdl-org/youtube-dl/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [ ] Checked the code with [flake8](https://pypi.python.org/pypi/flake8)

### In order to be accepted and merged into youtube-dl each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [x] Bug fix
- [ ] Improvement
- [ ] New extractor
- [ ] New feature

---

### Description of your *pull request* and other information

Related to https://github.com/ytdl-org/youtube-dl/pull/27234

In my limited test, the structure has become

```html
<div class="votes-fav-wrap">
	<div class="js-voteUp icon-wrapper tooltipTrig"
         data-title="顶一下"
            >
        <i class="thumbs-up"></i>
		<span class="votesUp" data-rating="4614">5K</span>
	</div>

	<div class="js-voteDown icon-wrapper tooltipTrig"
         data-title="踩一下"
            >
        <i class="thumbs-down"></i>
        <span class="votesDown" data-rating="703">703</span>
    </div>
</div>
```

### CODE REVIEW

**Comment by JChris246** [comment]
Yeah my bad, the video i tested with didn't have K in it. You should still keep the regex relaxed along with your changes

**Comment by imba-tjd** [comment]
IMO the relax doesn't make too much sense, because the data relies on `class="votesUp"`.

Or I'd prefer `class="votesUp"\s+data-rating="(\d+)"` i.e. remove `<span` if you insist the relax.

**Comment by JChris246** [comment]
i'm just quoting their doc on regex https://github.com/ytdl-org/youtube-dl#make-regular-expressions-relaxed-and-flexible 🤷‍♀️

**Comment by imba-tjd** [comment]
Yeah, I konw. I read and thought about it. Let the maintainer decide.

File: youtube_dl/extractor/pornhub.py [hunk 1]
```diff
@@ -354,9 +354,9 @@ def add_video_url(video_url):
         view_count = self._extract_count(
             r'<span class="count">([\d,\.]+)</span> [Vv]iews', webpage, 'view')
         like_count = self._extract_count(
-            r'<span[^>]+class="votesUp"[^>]*>([\d,\.]+)</span>', webpage, 'like')
+            r'<span class="votesUp" data-rating="(\d+)">', webpage, 'like')
```
Comment (imba-tjd) [inline youtube_dl/extractor/pornhub.py:357]:
```suggestion
            r'class="votesUp".+?data-rating="(\d+)"', webpage, 'like')
```

File: youtube_dl/extractor/pornhub.py [hunk 2]
```diff
@@ -354,9 +354,9 @@ def add_video_url(video_url):
...
         dislike_count = self._extract_count(
-            r'<span[^>]+class="votesDown"[^>]*>([\d,\.]+)</span>', webpage, 'dislike')
+            r'<span class="votesDown" data-rating="(\d+)">', webpage, 'dislike')
```
Comment (imba-tjd) [inline youtube_dl/extractor/pornhub.py:359]:
```suggestion
            r'class="votesDown".+?data-rating="(\d+)"', webpage, 'dislike')
```

**Review by dstftw — CHANGES_REQUESTED** [review]
1. Do not remove old patterns.
2. New patterns must be relaxed similarly.

File: youtube_dl/extractor/pornhub.py (hunk 1 above)
Comment (imba-tjd) [inline youtube_dl/extractor/pornhub.py:357]:
```suggestion
            r'<span[^>]+class="votesUp"[^>]+data-rating="(\d+)"[^>]*>', webpage, 'like')
```

File: youtube_dl/extractor/pornhub.py (hunk 2 above)
Comment (imba-tjd) [inline youtube_dl/extractor/pornhub.py:359]:
```suggestion
            r'<span[^>]+class="votesDown"[^>]+data-rating="(\d+)"[^>]*>', webpage, 'dislike')
```

**Pull request closed without merging**